src/content/docs/accounts/accounts/automated-user-management/automated-user-provisioning-single-sign.mdx

```diff
@@ -52,11 +52,9 @@ To use automated user management to import users from your identity provider:
 3. If you use Azure AD, Okta, or OneLogin, use the applicable guide: [Azure AD](/docs/azure-scimsso-application-configuration) \| [Okta](/docs/okta-scimsso-application-configuration) \| [OneLogin](/docs/onelogin-scimsso-application-configuration). 
 4. If you **don't** use one of the above services, you'll need to:
```

Comment:
The bullets following thec colon don't follow what I like to see there, that is the bullets completing the sentence started by the text before the colon.

For example, "If you don't use one of the above services, you'll need to: 
* Use the authentication domain UI to enable SCIM as the source of users."